sdk/formrecognizer/azure-ai-formrecognizer/tests/test_receipt_from_url.py

```diff
@@ -195,7 +180,6 @@ def test_receipt_url_jpg(self, client):
         receipt_type = receipt.fields.get("ReceiptType")
         self.assertIsNotNone(receipt_type.confidence)
         self.assertEqual(receipt_type.value, 'Itemized')
-        self.assertReceiptItemsHasValues(receipt.fields["Items"].value, receipt.page_range.first_page_number, False)
```

Comment:
I would keep this line, but delete assertFormPagesHasValues 4 lines above since we already call that directly after the poller result (same in async file)

Reply:
Done!